### PR TITLE
Added popup when listing is clicked

### PR DESCRIPTION
**Pull request by simazm8**

### CODE REVIEW

**Comment by simazm8** [comment]
As esu frontonedo loxas tai paziurekit ar gerai. Is principo kai paspaudi ant cardo tas popupas pasidaro visible. Gal geriausia butu kad ji tik sukurtu, bet mano net ir labai prastam kompe greit veike ir gerai. Reik pataisyt kad foto normaliai uzkeltu ir formatavima
Bet bent jau yra ant ko lipdyt

File: src/Components/ListingCard.js [hunk 1]
```diff
@@ -21,17 +24,44 @@ export const ListingCard = (props) => {
...
+    function ListingPopup(props) {
+        return (
+          <Modal {...props} size="lg">
+            <Modal.Header closeButton>
+              <Modal.Title id="contained-modal-title-vcenter">
+                {props.listing.title}
+              </Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+                {/**@TODO fix image cuz it's not working **/}
```
Comment (ElektrosStulpas) [inline src/Components/ListingCard.js:36]: image here would be nice to have, but not necessity

File: src/Components/ListingCard.js [hunk 2]
```diff
@@ -21,17 +24,44 @@ export const ListingCard = (props) => {
...
+                {/* <img>src={props.image} </img> */}
+                <p>{props.listing.description}</p>
+                <p>{props.listing.daysPrice}€ / day</p>
+            </Modal.Body>
+            <Modal.Footer>
+              {/** @TODO make this nicer cuz it looks uggo **/}
+              <p class="text-left">{props.city}</p>
+              <Button>Rent</Button>
```
Comment (ElektrosStulpas) [inline src/Components/ListingCard.js:44]: Does this button do something?

File: src/Components/ListingCard.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import axios from "axios";
 import React, { useEffect, useState } from "react"
 import { Card } from 'react-bootstrap';
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
 
 export const ListingCard = (props) => {
+    const [modalShow, setModalShow] = React.useState(false);
```
Comment (ElektrosStulpas) [inline src/Components/ListingCard.js:8]: since useState is imported already, don't need to write React.useState, can just do useState

**Review by ElektrosStulpas — COMMENTED** [review]
bit barebones, don't know if you can actually rent it or if it's just a button. Can merge just to have as much as fast on main, but having actual renting capability beforehand would be awesome

**Pull request merged**